ksql-common/src/main/java/io/confluent/ksql/config/ConfigItem.java

```diff
@@ -22,6 +22,11 @@
 
 public interface ConfigItem {
 
+  /**
+   * @return {@code true} is this is a known, resolved config.
+   */
+  boolean isResolved();
```

Comment:
should we default this to `false`?

Reply:
It's only got two impls - so makes little difference IMHO,